packages/documentation/src/stories/components/forms/slider/slider.stories.ts

```diff
@@ -202,7 +202,7 @@ function render(args: Args, context: StoryContext) {
   let valueElement: TemplateResult | TemplateResult[] | null = null;
 
   if (args.showValue === 'text') {
-    valueElement = html` <p class="form-text">${args.value}</p> `;
+    valueElement = html` <p class="form-hint">${args.value}</p> `;
```

Comment:
Also connect this on the input element via aria-describedby.

```suggestion
    valueElement = html` <p class="form-hint" id="form-hint-example">${args.value}</p> `;
```